src/reducers/filter.js

```diff
@@ -1,10 +1,12 @@
-const filterReducer = (state = [], action) => {
+const filterReducer = (state = 'All', action) => {
   switch (action.type) {
     case 'CHANGE_FILTER':
-      return state.filter((book) => book.category === action.book.category);
+      return action.filter;
     default:
       return state;
   }
 };
 
 export default filterReducer;
+
+// ((book) => book.category !== action.filter.category);
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

I trust you can do this before merging, no need for a new CRR for this commented code, 👀 on you.